kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/commands/KieServerControllerDescriptorCommand.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.controller.api.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElements;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.kie.server.api.model.KieServerCommand;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.controller.api.model.runtime.ServerInstanceKey;
+import org.kie.server.controller.api.model.spec.*;
+
+@XmlRootElement(name = "controller-descriptor-command")
+@XmlAccessorType(XmlAccessType.NONE)
+public class KieServerControllerDescriptorCommand implements KieServerCommand {
```

Comment:
What is the purpose of this class?

Reply:
@sutaakar this will be used in the websocket client management api, I added in here because it was needed for completely testing the serialization issue I was having.

Reply:
Ok, thanks for info.